app/src/download.js

```diff
@@ -208,11 +208,14 @@
 			if ( !thisName || !thisDependencies ) {
 				return;
 			}
-			thisDependencies = thisDependencies.split( "," );
+			thisDependencies = thisDependencies.split( "," ).map(function( dependency ) {
+				// remove any path prefixes, ain't caring about those
+				return dependency.replace( /^.+\//, '' );
```

Comment:
This is an incomplete fix to deal with the new folder structure. Widget-to-widget dependencies are still broken.